Tests/ShortcutFoundationTests/IntTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  IntTests.swift
+//  ShortcutFoundation
+//
+//  Created by Andreas Lif on 2022-01-12.
+//  Copyright © 2021 Shortcut Scandinavia Apps AB. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import ShortcutFoundation
+
+final class IntTests: XCTestCase {
+    
+    let odds = [1, 23, 405, 6897]
```

Comment:
You can add the odds array to the odds unit tests

Reply:
@gabrielPeart Thanks! That's how I originally did it, then I tested something and forgot to put it back ☺️ Fixed in 37f1d85

Reply:
Changed again, to your suggestion, in 5fcbeba